.stylelintrc

```diff
@@ -1,119 +1,10 @@
 {
-  "extends": ["stylelint-config-recommended", "stylelint-config-idiomatic-order"],
-  "plugins": [
-    "stylelint-scss"
-  ],
+  "extends": "stylelint-config-recommended",
```

Comment:
This is wrong. See https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/.stylelintrc